src/experimental/constructs/error-budget-alarm.test.ts

```diff
@@ -16,6 +20,66 @@ describe("The ErrorBudgetAlarmExperimental construct", () => {
     expect(Template.fromStack(stack).toJSON()).toMatchSnapshot();
   });
 
+  it("should create the correct resources for a simple Availability SLO", () => {
+    const stack = simpleGuStackForTesting();
+    const ec2App = new GuEc2App(stack, {
+      applicationPort: 3000,
+      app: "test-gu-ec2-app",
+      access: { scope: AccessScope.PUBLIC },
+      instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.MEDIUM),
+      monitoringConfiguration: { noMonitoring: true },
+      userData: "#!/bin/dev foobarbaz",
+      certificateProps: {
+        domainName: "code-guardian.com",
+      },
+      scaling: {
+        minimumInstances: 1,
+      },
+    });
+    new GuErrorBudgetAlarmExperimental(stack, {
```

Comment:
If desired, we could extend this class in the future (e.g. `GuErrorBudgetAvailabilityAlarmExperimental` and `GuErrorBudgetLatencyAlarmExperimental`) to pre-populate these metrics for these common SLI types.

What do you think @michaelwmcnamara and @jorgeazevedo?

Reply:
Yes! We can essentially document/standardise how to compute the bad events/valid events for these known cases like elb availability and latency by providing those classes